src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -209,6 +209,9 @@ class MediaBaseViewer extends BaseViewer {
         this.emit(VIEWER_EVENT.load);
 
         this.loadUI();
+        // If media is muted because of auto-play,
+        // the volume icon should be updated after the UI is loaded
+        this.updateVolumeIcon();
```

Comment:
Should this call live within `handleVolume`?

Reply:
yeah, but `handleVolume` is called within `autoplay`, which happens before `loadUI`, so we have to call `updateVolumnIcon` explicitly again.

Reply:
Another way to do this is to move `autoplay` call after `loadUI`. What do you think?

Reply:
Sure It seems like DashViewer and MediaBaseViewer should be as similar as possible in this regard.